modules/app.js

```diff
@@ -0,0 +1,94 @@
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+const Store = {
+  getBooks() {
+    let bookInputs;
+    if (localStorage.getItem('bookInputs') === null) {
+      bookInputs = [];
+    } else {
+      bookInputs = JSON.parse(localStorage.getItem('bookInputs'));
+    }
+    return bookInputs;
+  },
+
+  addBook(book) {
+    const bookInputs = Store.getBooks();
+    bookInputs.push(book);
+    localStorage.setItem('bookInputs', JSON.stringify(bookInputs));
+  },
+
+  removeBook(del) {
+    const bookInputs = Store.getBooks();
+    bookInputs.forEach((book, index) => {
+      if (book.del === del) {
+        bookInputs.splice(index, 1);
+      }
+    });
+
+    localStorage.setItem('bookInputs', JSON.stringify(bookInputs));
+  },
+};
+
+const page = {
+  displayBooks() {
+    const newBooks = Store.getBooks();
+    newBooks.forEach((books) => page.bookList(books));
+  },
+
+  bookList(book) {
+    const list = document.getElementById('book-collection');
+    const create = document.createElement('div');
+    create.className = 'list-control';
+
+    const p = document.createElement('p');
+    const b = document.createElement('b');
+    b.textContent = ' By ';
+    p.append(book.title, b, book.author);
+
+    const btn = document.createElement('button');
+    btn.classList.add('remove-btn');
+    btn.textContent = 'Remove';
+
+    create.append(p, btn);
+    list.appendChild(create);
+  },
+
+  deleteBook(target) {
+    if (target.classList.contains('remove-btn')) {
+      target.parentElement.remove();
+    }
+  },
+};
+
+document.querySelector('.book-input').addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  if (title && author) {
+    const book = new Book(title, author);
+    page.bookList(book);
+    Store.addBook(book);
+  }
+
+  // to clear fields
+  e.target.reset();
+});
+
+// to remove a book
+document.querySelector('#book-collection').addEventListener('click', (e) => {
+  page.deleteBook(e.target);
+  Store.removeBook();
+});
+
+(function render() {
+  const books = Store.getBooks();
+  books.forEach((book) => {
+    page.bookList(book);
+  });
+}());
```

Comment:
- [ ] Kindly adherent to the suggestion by the previous reviewer regarding practising modular programming. :point_down: 
`I would suggest that you continue to break up your codes located here in index.js file so that you exercise more with modular structuring smiley so please make sure that you group some of the functions here in this file into its separate file in the modules folder wink`.
This file still big and can still be split into smaller files. Kindly do that please

Reply:
These codes are not in my index.js file they are in a separate file called app.js
![Screenshot 2022-11-01 at 7 40 53 AM](https://user-images.githubusercontent.com/105528809/199174505-c661c1d4-d9e7-4f6a-9c21-b711be87cc06.png)

How else do you want me to break them?